sklearn/datasets/tests/test_base.py

```diff
@@ -252,6 +253,18 @@ def test_loads_dumps_bunch():
     assert bunch_from_pkl['x'] == bunch_from_pkl.x
 
 
+def test_fetch_asframe():
```

Comment:
the test should be in test_california_housing.py I think.

Reply:
Gotcha; thanks!  Wasn't sure since it aims to test a function in the `_base.py` file.  I'll move it now.